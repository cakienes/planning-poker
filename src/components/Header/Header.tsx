import React from 'react';
import PlanningPoker from './PlanningPoker.jpg';

const Header: React.FC<{}> = () => {
    return (
        <div className="mt-3 mb-3">
            <img src={PlanningPoker} alt="Scrum Poker" />
        </div>
    );
};

export default Header;
