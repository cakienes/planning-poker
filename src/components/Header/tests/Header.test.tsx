import React from 'react';
import { createRenderer, ShallowRenderer } from 'react-test-renderer/shallow';
import Header from '../Header';

describe('Header.tsx', () => {
    let renderer: ShallowRenderer;
    beforeEach(() => {
        renderer = createRenderer();
    });

    it('render correctly Header', async () => {
        expect(renderer.render(<Header />)).toMatchSnapshot();
    });
});
