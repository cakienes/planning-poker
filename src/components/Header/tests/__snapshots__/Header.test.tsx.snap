// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.tsx render correctly Header 1`] = `
<div
  className="mt-3 mb-3"
>
  <img
    alt="Scrum Poker"
    src="PlanningPoker.jpg"
  />
</div>
`;
